Stripe/STPFile.m

```diff
@@ -11,16 +11,18 @@
 
 #import "NSDictionary+Stripe.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface STPFile ()
 
-@property (nonatomic, readwrite) NSString *fileId;
-@property (nonatomic, readwrite) NSDate *created;
-@property (nonatomic, readwrite) STPFilePurpose purpose;
-@property (nonatomic, readwrite) NSNumber *size;
-@property (nonatomic, readwrite) NSString *type;
-@property (nonatomic, readwrite, copy) NSDictionary *allResponseFields;
+@property (nonatomic, copy, readwrite) NSString *fileId;
+@property (nonatomic, copy, readwrite) NSDate *created;
+@property (nonatomic, assign, readwrite) STPFilePurpose purpose;
+@property (nonatomic, copy, readwrite) NSNumber *size;
+@property (nonatomic, copy, readwrite) NSString *type;
+@property (nonatomic, copy, readwrite, copy) NSDictionary *allResponseFields;
```

Comment:
Just glancing at this PR again, we should list the classes that should be `copy` instead of `strong` in the style guide. Currently it's just:

- NSString
- NSDictionary
- NSArray

Should we add NSDate and NSNumber? or is this just a copy/paste bug?